lab_7_llm/tests/metric_check_test.py

```diff
@@ -3,7 +3,7 @@
 """
```

Comment:
Checkout the files from 7 lab to the master

Reply:
All lab_7 files from the main branch are identical to files in my fork.